filters/auth/tokenintrospection.go

```diff
@@ -0,0 +1,342 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	OAuthTokenintrospectionAnyClaimsName = "oauthTokenintrospectionAnyClaims"
+	OAuthTokenintrospectionAllClaimsName = "oauthTokenintrospectionAllClaims"
+	OAuthTokenintrospectionAnyKVName     = "oauthTokenintrospectionAnyKV"
+	OAuthTokenintrospectionAllKVName     = "oauthTokenintrospectionAllKV"
+
+	tokenintrospectionCacheKey   = "tokenintrospection"
+	TokenIntrospectionConfigPath = "/.well-known/openid-configuration"
+)
+
+type (
+	tokenIntrospectionSpec struct {
+		typ              roleCheckType
+		issuerURL        string
+		introspectionURL string
+		config           *OpenIDConfig
+		timeout          time.Duration
+		authClient       *authClient
```

Comment:
is the `authClient` field of the filter spec used?

Reply:
seems to be a leftover